src/components/TasksTable/index.jsx

```diff
@@ -327,6 +383,28 @@ export default class TasksTable extends Component {
                 </MenuItem>
               ))}
             </TextField>
+            <TextField
+              select
+              label="Sort by"
+              value={sortBy || HEADERS.LAST_UPDATED}
+              className={classes.dropdown}
+              onChange={this.handleSortBy}>
+              {headers.filter(header => header !== HEADERS.TAGS).map(header => (
+                <MenuItem key={header} value={header}>
+                  {header === sortBy &&
+                    (sortDirection === 'asc' ? (
+                      <ArrowUpIcon size={iconSize} />
+                    ) : (
+                      <ArrowDownIcon size={iconSize} />
+                    ))}
+                  {sortBy &&
+                    header !== sortBy && (
+                      <Fragment>&nbsp;&nbsp;&nbsp;&nbsp;</Fragment>
+                    )}
+                  {header}
+                </MenuItem>
```

Comment:
![screen shot 2018-07-16 at 9 19 55 am](https://user-images.githubusercontent.com/3766511/42760800-9435da28-88d9-11e8-93f4-f4a6956c3702.png)

* The arrow should be a little bit taller
* There should be a little bit more space between the text and the arrow icon